daemon/emer-daemon.c

```diff
@@ -1255,6 +1255,14 @@ set_max_bytes_buffered (EmerDaemon *self,
   priv->max_bytes_buffered = max_bytes_buffered;
 }
 
+static gchar *
+unhyphenate_uuid (gchar *uuid_with_hyphens)
+{
+  return g_strdup_printf ("%.8s%.4s%.4s%.4s%.12s\n", uuid_with_hyphens,
+                          uuid_with_hyphens + 9, uuid_with_hyphens + 14,
+                          uuid_with_hyphens + 19, uuid_with_hyphens + 24);
+}
+
```

Comment:
I recognise this code from the tests. Perhaps it could live in metrics-shared.[ch]?